shell/common/engine.h

```diff
@@ -69,59 +274,429 @@ class Engine final : public RuntimeDelegate {
 
+  //----------------------------------------------------------------------------
+  /// @brief      Updates the viewport metrics for the currently running Flutter
+  ///             application. The viewport metrics detail the size of the
+  ///             rendering viewport in texels as well as edge insets if
+  ///             present.
+  ///
+  /// @see        `ViewportMetrics`
+  ///
+  /// @param[in]  metrics  The metrics
+  ///
   void SetViewportMetrics(const ViewportMetrics& metrics);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Notifies the engine that the embedder has sent it a message.
+  ///             This call originates in the platform view and has been
+  ///             forwarded to the engine on the UI task runner here.
+  ///
+  /// @param[in]  message  The message sent from the embedder to the Dart
+  ///                      application.
+  ///
   void DispatchPlatformMessage(fml::RefPtr<PlatformMessage> message);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Notifies the engine that the embedder has sent it a pointer
+  ///             data packet. A pointer data packet may contain multiple
+  ///             input events. This call originates in the platform view and
+  ///             the shell has forwarded the same to the engine on the UI task
+  ///             runner here.
+  ///
+  /// @param[in]  packet         The pointer data packet containing multiple
+  ///                            input events.
+  /// @param[in]  trace_flow_id  The trace flow identifier associated with the
+  ///                            pointer data packet. The engine uses this trace
+  ///                            identifier to connect trace flows in the
+  ///                            timeline from the input event event to the
+  ///                            frames generated due to those input events.
+  ///                            These flows are tagged as "PointerEvent" in the
+  ///                            timeline and allow grouping frames and input
+  ///                            events into logical chunks.
+  ///
   void DispatchPointerDataPacket(const PointerDataPacket& packet,
                                  uint64_t trace_flow_id);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Notifies the engine that the embedder encountered an
+  ///             accessibility related action on the specified node. This call
+  ///             originates on the platform view and has been forwarded to the
+  ///             engine here on the UI task runner by the shell.
+  ///
+  /// @param[in]  id      The identifier of the accessibility node.
+  /// @param[in]  action  The accessibility related action performed on the
+  ///                     node of the specified ID.
+  /// @param[in]  args    An optional list of argument that apply to the
```

Comment:
arguments

Though really it's just whatever arbitrary blob of serialised data we decided to associate with the action, so maybe just "Optional data that applies to the specified action."

Reply:
Done.